src/App.css

```diff
@@ -6,15 +6,23 @@
   align-items: center;
 }
 
+/* Nav header  */
 .logo-container {
   display: flex;
   align-items: center;
+  margin-top: 10px;
+  margin-left: 17px;
 }
 
 .app-logo {
```

Comment:
You updated your CSS file before pulling from dev branch. you dev branch is updated with rockets-list branch from your coding partners side